src/test/java/mate/academy/dao/impl/RoleDaoTest.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.dao.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import mate.academy.dao.RoleDao;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoTest extends AbstractTest {
+    private static final String ADMIN = Role.RoleName.ADMIN.name();
+    private static final String isNotPresentRole = "USR";
```

Comment:
```suggestion
    private static final String notPresentRole = "USR";
```

зазвичай приставка `is` використовується для boolean змінних